www/js/lib/util.js

```diff
@@ -78,16 +78,16 @@ define(['q'], function(q) {
     }
     
     /**
-     * Generates an array of Titles, where all duplicates have been removed
-     * (it also sorts the titles)
+     * Generates an array of DirEntry, where all duplicates (same title) have been removed
+     * (it also sorts them on their title)
      * 
-     * @param {Array.<Title>} array of Titles
-     * @returns {Array.<Title>} same array of Titles, without duplicates
+     * @param {Array.<DirEntry>} array of DirEntry
+     * @returns {Array.<DirEntry>} same array of DirEntry, without duplicates
      */
-    function removeDuplicateTitlesInArray(array) {
-        array.sort(function(titleA, titleB) {
-            if (titleA.title < titleB.title) return -1;
-            if (titleA.title > titleB.title) return 1;
+    function removeDuplicateTitlesInDirEntryArray(array) {
+        array.sort(function(dirEntryA, dirEntryB) {
+            if (dirEntryA.title < dirEntryB.title) return -1;
+            if (dirEntryA.title > dirEntryB.title) return 1;
```

Comment:
I agree. It was probably meaningful with Evopedia archives, but is not any more.